platforms/Linux/DEB/Ubuntu/focal/debian/control.in

```diff
@@ -0,0 +1,46 @@
+Source: swiftlang
+Section: devel
+Priority: optional
+Maintainer: Swift Maintainers <swift-infrastructure@forums.swift.org>
```

Comment:
is there a way to read this from a separate file?

Reply:
The `control` file is monolithic and doesn't allow for includes or similar. Some fields for binary packages can be filled in during build via debhelper substitutions, but for the source package, that's not the case.